lib/utils/format_utils.js

```diff
@@ -62,7 +62,8 @@ function dateFormatForLiquibase(args) {
     args.date = new Date(args.date.getTime());
   }
   const merged = merge(defaultsForLiquibaseDateFormatting(), args);
-  merged.date.setSeconds(merged.date.getUTCSeconds() + merged.increment);
+  merged.date.setMinutes(merged.date.getMinutes() + merged.increment);
```

Comment:
why?

Reply:
Used minutes at generator-jhipster patch. 
https://github.com/jhipster/generator-jhipster/blob/1db47508bb9a89d16ef5798bee0ee9c63c9300a1/generators/generator-base.js#L822
This will keep a delta for custom changelogs.

Removed UTC because getUTC is already used next.
https://github.com/jhipster/jhipster-core/blob/d3193b5d5c7d2e66eb75fe9ef34f3fb5e8d106b5/lib/utils/format_utils.js#L72

Running getUTCSeconds from a date with seconds set with a getUTCSeconds will duplicate conversion if it exists.

Reply:
fair enough